main.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-unused-vars, no-use-before-define */
+
+const bookArray = localStorage.getItem('books') ? JSON.parse(localStorage.getItem('books')) : [];
+localStorage.setItem('books', JSON.stringify(bookArray));
+const library = JSON.parse(localStorage.getItem('books'));
+
+function Book(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+function createList(title, author) {
+  const li = document.createElement('li');
+  li.textContent = `${title} - by ${author}`;
+  const button = document.createElement('button');
+  button.setAttribute('class', 'rmItem');
+  button.innerHTML = 'Remove';
+  button.addEventListener('click', remove);
+  li.appendChild(button);
+  const ul = document.getElementById('bookList');
+  ul.appendChild(li);
+}
```

Comment:
- [ ]  Kindly ensure `arrow functions` are used instead of the `function keyword`